languages/index.js

```diff
@@ -1,730 +1,736 @@
-    scopeName: 'source.shaderlab',
-    grammar: loadJSON('shaderlab.tmLanguage.json'),
-    aliases: ['shader']
-  },
-  {
-    name: 'Shell',
-    id: 'shellscript',
-    scopeName: 'source.shell',
-    grammar: loadJSON('shellscript.tmLanguage.json'),
-    aliases: ['shell', 'bash', 'sh', 'zsh']
-  },
-  {
-    name: 'Smalltalk',
-    id: 'smalltalk',
-    scopeName: 'source.smalltalk',
-    grammar: loadJSON('smalltalk.tmLanguage.json')
-  },
-  {
-    name: 'SQL',
-    id: 'sql',
-    scopeName: 'source.sql',
-    grammar: loadJSON('sql.tmLanguage.json')
-  },
-  {
-    name: 'SSH Config',
-    id: 'ssh-config',
-    scopeName: 'source.ssh-config',
-    grammar: loadJSON('ssh-config.tmLanguage.json')
-  },
-  {
-    name: 'Stylus',
-    id: 'stylus',
-    scopeName: 'source.stylus',
-    grammar: loadJSON('stylus.tmLanguage.json'),
-    aliases: ['styl']
-  },
-  {
-    name: 'Swift',
-    id: 'swift',
-    scopeName: 'source.swift',
-    grammar: loadJSON('swift.tmLanguage.json')
-  },
-  {
-    name: 'Tcl',
-    id: 'tcl',
-    scopeName: 'source.tcl',
-    grammar: loadJSON('tcl.tmLanguage.json')
-  },
-  {
-    name: 'TOML',
-    id: 'toml',
-    scopeName: 'source.toml',
-    grammar: loadJSON('toml.tmLanguage.json')
-  },
-  {
-    name: 'TypeScript Doc',
-    id: 'ts-doc',
-    scopeName: 'documentation.injection.ts',
-    grammar: loadJSON('ts.tmLanguage.json')
-  },
-  {
-    name: 'TSX',
-    id: 'tsx',
-    scopeName: 'source.tsx',
-    grammar: loadJSON('tsx.tmLanguage.json')
-  },
-  {
-    name: 'TypeScript',
-    id: 'typescript',
-    scopeName: 'source.ts',
-    grammar: loadJSON('typescript.tmLanguage.json'),
-    aliases: ['ts']
-  },
-  {
-    name: 'Visual Basic .NET',
-    id: 'vb',
-    scopeName: 'source.asp.vb.net',
-    grammar: loadJSON('vb.tmLanguage.json'),
-    aliases: ['cmd']
-  },
-  {
-    name: 'VimL',
-    id: 'viml',
-    scopeName: 'source.viml',
-    grammar: loadJSON('viml.tmLanguage.json'),
-    aliases: ['vim']
-  },
-  {
-    name: 'Vue HTML',
-    id: 'vue-html',
-    scopeName: 'text.html.vue-html',
-    grammar: loadJSON('vue-html.tmLanguage.json')
-  },
-  {
-    name: 'Vue',
-    id: 'vue',
-    scopeName: 'source.vue',
-    grammar: loadJSON('vue.tmLanguage.json')
-  },
-  {
-    name: 'WebAssembly',
-    id: 'wasm',
-    scopeName: 'source.wat',
-    grammar: loadJSON('wasm.tmLanguage.json'),
-    aliases: ['wat']
-  },
-  {
-    name: 'XML',
-    id: 'xml',
-    scopeName: 'text.xml',
-    grammar: loadJSON('xml.tmLanguage.json')
-  },
-  {
-    name: 'XSL',
-    id: 'xsl',
-    scopeName: 'text.xml.xsl',
-    grammar: loadJSON('xsl.tmLanguage.json')
-  },
-  {
-    name: 'YAML',
-    id: 'yaml',
-    scopeName: 'source.yaml',
-    grammar: loadJSON('yaml.tmLanguage.json'),
-    aliases: ['yml']
-  },
-  {
-    name: 'Wenyan',
-    id: '文言',
-    scopeName: 'source.wenyan',
-    grammar: loadJSON('文言.tmLanguage.json'),
-    aliases: ['wenyan']
-  }
-]
+const fs = require('fs')

```

Comment:
Your IDE made the whole file CRLF line endings, can you revert this back to LF before I merge

Reply:
Changed to LF